extension/ui/components/input_bar/editor/useCustomEditor.tsx

```diff
@@ -8,9 +8,9 @@ import { URLStorageExtension } from "@app/ui/components/input_bar/editor/extensi
 import { createMarkdownSerializer } from "@app/ui/components/input_bar/editor/markdownSerializer";
 import type { EditorSuggestions } from "@app/ui/components/input_bar/editor/suggestion";
 import type { SuggestionProps } from "@app/ui/components/input_bar/editor/useMentionDropdown";
-import { MentionPluginKey } from "@tiptap/extension-mention";
 import Paragraph from "@tiptap/extension-paragraph";
 import Placeholder from "@tiptap/extension-placeholder";
+import { PluginKey } from "@tiptap/pm/state";
```

Comment:
it's part of the v2 upgrade I do in the extension at the same time (already applied in front)